howto/manage/benchmarks.md

```diff
@@ -1,27 +1,47 @@
 
-[note type="caution" status="Warning"]If your application is not constantly refreshing the screen by itself, like regular Android applications providing a simple user interface do, you will get a low FPS number. The benchmark can only provide useful information if you know your workload and how it should perform in an ideal scenario.[/note]
+## Run container benchmarks
 
-### Run the Benchmark
+The `amc` command line utility provides a `benchmark` subcommand to run benchmarks on an Anbox Cloud deployment. It measures the time containers take to start up and their average FPS (frames per second) rate.
 
-Have a look at the output of
+The results you get depend on the resources of the machines that host the containers. In addition, latency between the different nodes and the AMS services can impact the results
 
-```bash
-$ amc benchmark -h
-```
+Check the help output to see all available arguments and their purpose:
 
-to learn more about the different arguments and their purpose.
+    amc benchmark -h
 
-[note type="information" status="Hint"]Keep in mind that the success of container startup and the times you will get depend on  the underlying resources used on the machines hosting the containers. Latency between the different nodes and the AMS services also play their role.[/note]
+The benchmark command launches the specified number of containers on the Anbox [`null` platform](https://discourse.ubuntu.com/t/anbox-platforms/18733) with the following default display specification:
 
-An example benchmark session looks like this:
+ | Display spec            | Value |
+ | ----------------------- | ----- |
+ | Display width           | 1280  |
+ | Display height          | 720   |
+ | FPS                     | 60    |
+ | Display density         | 160   |
+
+You can configure a different display specification through the `--user-data` parameter when running the benchmark. The required format for the parameter varies based on the different platforms:
+
+| Platform              | Required format of user data          | Example                                                   |
+| --------------------- | ------------------------------------  | --------------------------------------------------------  |
+| Null                  | Comma-separated values                | <display_width>,<display_height>,<display_fps>,<display_density>  |
+| Swrast                | Comma-separated values                | <display_width>,<display_height>,<display_fps>,<display_density>  |
+| WebRTC                | JSON-based                            | {<br>"display_width": <display_width>,<br>"display_height": <display_height>,<br>"display_density": <display_density>,<br>"fps": <display_fps>,<br>"render_only": true<br> } |
```

Comment:
We should name the platforms by their real name, e.g. "webrtc", "swrast" and "null"